src/libsync/theme.cpp

```diff
@@ -533,7 +549,22 @@ QColor Theme::wizardHeaderBackgroundColor() const
 QPixmap Theme::wizardHeaderLogo() const
 {
 #ifdef APPLICATION_WIZARD_USE_CUSTOM_LOGO
-   return QPixmap(hidpiFileName(":/client/theme/colored/wizard_logo.png"));
+#ifdef APPLICATION_ICON_SET
+    const auto useSvg = QByteArray(APPLICATION_ICON_SET).toUpper() == QByteArrayLiteral("SVG");
+#else
+    const auto useSvg = false;
+#endif
+
+    if (useSvg) {
+        const auto maxHeight = 64;
+        const auto maxWidth = 2 * maxHeight;
+        const auto icon = QIcon(":/client/theme/colored/wizard_logo.svg");
+        const auto size = icon.actualSize(QSize(maxWidth, maxHeight));
+        const auto pixmap = icon.pixmap(size);
```

Comment:
@er-vin I'd get rid of this rather redundant **pixmap ** variable.

Reply:
done